File: client/java/src/avt/RoomInfo.java
package avt;

public final class RoomInfo {
   public byte id;
   public byte roomFree;
   public byte lv;

   public RoomInfo(byte var1, byte var2, byte var3, byte var4) {
      this.id = var1;
      this.roomFree = 0;
      this.lv = var4;
   }

   public RoomInfo() {
   }
}

File: client/unity/Assets/Scripts/Assembly-CSharp/RoomInfo.cs
public class RoomInfo
{
	public sbyte id;

	public sbyte roomFree;

	public sbyte roomWait;

	public sbyte lv;

	public RoomInfo(sbyte id, sbyte roomFree, sbyte roomWait, sbyte lv)
	{
		this.id = id;
		this.roomFree = roomFree;
		this.roomWait = roomWait;
		this.lv = lv;
	}

	public RoomInfo()
	{
	}
}
